HLTrigger/HLTcore/plugins/HLTEventAnalyzerRAW.cc

```diff
@@ -37,7 +37,7 @@
 #include "DataFormats/L1Trigger/interface/Tau.h"
 #include "DataFormats/L1Trigger/interface/EtSum.h"
 
-#include "DataFormats/L1TCorrelator/interface/TkMuon.h"
+#include "DataFormats/L1TMuonPhase2/interface/TrackerMuon.h"
```

Comment:
`DataFormats/L1TMuonPhase2` needs to be added to the `plugins/BuildFile.xml` file.

Reply:
Agree, to be added.